src/main/java/marquez/api/models/JobsResponse.java

```diff
@@ -17,12 +17,12 @@
 import java.util.List;
 import lombok.AllArgsConstructor;
 import lombok.Data;
-import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.NonNull;
 
 @Data
-@NoArgsConstructor
 @AllArgsConstructor
+@NoArgsConstructor
```

Comment:
We should only annotate this class with `@Value`

Reply:
Unless you added the additional annotations to get the tests to pass?

Reply:
yes, test cases were failing, so I had added these annotations. I would review the failing test cases, so as to handle `@value` annotation

Reply:
@wslulciuc, updated the classes to use `@Value` annotation, and fixed failing test cases.